com.unity.netcode.gameobjects/Runtime/Transports/MultiplexTransportAdapter.cs

```diff
@@ -1,329 +0,0 @@
-using System;
```

Comment:
Ok...is this part of removing channels?

Reply:
err... no but we're not planning to support this. could move under a separate PR to remove (didn't want to fix channels stuff in this file since we'd remove it right after already)